### DISCUSSION TITLE
Allowing <podcast:socialInteract> in the <channel>?

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#601] [Enhancement Proposal]
If a podcast has a social media account, it would be nice to be able to set that at the <channel> level so that it can pass through to clients.  I'm building the PI-AP bridge so this is something I want for that purpose.  But, as developers start to leverage ActivityPub/BlueSky/Nostr/etc more, it would be good to have that available.

I'm thinking it could be something like:

```xml
<podcast:socialInteract 
    protocol="activitypub" 
    uri="https://social.podnews.net/@podnews/110401010985487762" 
    accountId="@podnews@podnews.social.net" 
    accountUrl="https://social.podnews.net/@podnews"
/>
```

The `uri` in this case would be a link to a sticky post perhaps, or some other post that is a good intro post.  Or whatever works for the podcaster.  The `accountId` and `accountUrl` would be the most important thing for discoverability.

**Comment by johnspurlock** [comment]
I believe @benjaminbellamy had something like this as part of his initial spec: https://github.com/Podcastindex-org/podcast-namespace/blob/main/proposal-docs/social/social.md

**Reply by daveajones** [reply]
Yes, I remember this now, thanks.  In his proposal it was a different tag: `<podcast:social>` that identified things at that level.  I'm not sure we need a new tag for that.  Thoughts?

**Comment by nathangathright** [comment]
Could you clarify why it would require an `accountUrl`, a `URI` and a `accountId`? That feels redundant without whatever context I’m missing.

**Reply by daveajones** [reply]
They were required in Ben's proposal, but not in the `<podcast:socialInteract>` tag final spec:

![image](https://github.com/Podcastindex-org/podcast-namespace/assets/1498236/f004474c-c540-4f6f-8f8d-fefbb603ae00)

If we stick with just allowing `<podcast:socialInteract>` in the `<channel>` it would come with those requirements intact, which seems ok to me.  But, it actually doesn't address my intended use case here.  

I think this should probably be adopted, but my initial idea isn't really what this solves, which is fine.

**Comment by jamescridland** [comment]
I'd support this - and, indeed, would expect this to be the default for most podcasts that don't get enough chatter to justify one in the `<item>`
